<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-138</y>
        <width>866</width>
        <height>718</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>700</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(179, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Налаштування</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Управління Дудлом</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="radioButton_2">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>&lt;- (вліво) та -&gt; (вправо)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QRadioButton" name="radioButton">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>A(вліво) та D(вправо)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Фон гри</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="radioButton_6">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Звичайний</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QRadioButton" name="radioButton_8">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Вогонь</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QRadioButton" name="radioButton_7">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Флаг України</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Скін дудла</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QRadioButton" name="radioButton_3">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Ніндзя</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QRadioButton" name="radioButton_4">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Заєць</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QRadioButton" name="radioButton_5">
            <property name="styleSheet">
             <string notr="true">font: 900 10pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Контурний</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
